src/Writer/Drupal.php

```diff
@@ -35,8 +35,9 @@ public function rewrite($version, $timestamp)
         foreach ($this->paths as $info_file) {
             // Don't write to files that already contain version information.
             if (!$this->hasVersionInfo($info_file)) {
+                $project = basename(dirname($info_file));
```

Comment:
Instead of the directory name, shouldn't this be the name of the `.info` file, since they can technically be different (in D7 anyway). A good example of that is the [Google Analytics](https://www.drupal.org/project/google_analytics) module which uses `google_analytics` for the directory name but `googleanalytics` for the project name.